SH4RE/SH4RE/Home/HomeView.swift

```diff
@@ -125,6 +127,15 @@ struct HomeView: View {
                 fetchMainImage(listing: listing2, completion: { image2 in
                     recentListing2Image = image2
                 })
+                
+                chatLogViewModel1 = ChatLogViewModel(chatUser: ChatUser(id:listing1.uid, uid: listing1.uid, name: listing1.ownerName))
+                getProfilePic(uid: listing1.uid, completion: { profilePic in
+                    chatLogViewModel1?.profilePic = profilePic
+                })
+                chatLogViewModel2 = ChatLogViewModel(chatUser: ChatUser(id:listing2.uid, uid: listing2.uid, name: listing2.ownerName))
```

Comment:
why do you have to get the profile pics on home view? isn't this something that would happen in viewlistingview or messages or profile view?

Reply:
The way messaging is designed we need to instantiate a chatUser at the time we query the listing. So these are just creating the chatUser for the 2 recent listings